packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -114,9 +114,9 @@ class BuildInfo {
   /// rerun tasks.
   final String performanceMeasurementFile;
 
-  /// If provided, an output file where a v8-style heapsnapshot will be written for size
-  /// profiling.
-  final String analyzeSize;
+  /// If provided, an output directory where one or more v8-style heapsnapshots
```

Comment:
what about apk-analysis.json files? Additionally, if we are storing the apk-analysis.json files, why do we need to also store the v8 snapshots? An apk-analysis file will contain the v8 snapshot information

Reply:
See question above on the format.

Reply:
I'm not quite sure of the format of the apk-analysis file, seems like:

1. It needs to contain the snapshot, the path to libflutter.so, and some snapshot type key - right?

Should probably be called something else for the other platforms: apk-analysis.arm64.json, ios-analysis.json, et cetera

Reply:
The apk-analysis.json file currently contains the unzipped APK data, processed into a json format that DevTools uses to parse AOT snapshot trees. At the libapp.so node (defaulting to only one ABI currently), we swap in processed v8 snapshot data. The ending result is that the apk-analysis.json file contains a tree structure of the apk where the AOT snapshot has been embedded into the default libapp.so node.

The "type" key, is do differentiate apk-analysis files from v8 snapshot files when inputting them into DevTools. I think if would be fine if the value of "type" was the target platform ("android", "ios", "macos", "linux", "windows")